tests/core/test_graph.py

```diff
@@ -762,7 +762,7 @@ def f():
 @pytest.mark.parametrize("num_nodes,num_edges", [(0, 0), (100, 200), (1000, 5000)])
 # features or not, to capture their cost
 @pytest.mark.parametrize("feature_size", [None, 100])
-@pytest.mark.parametrize("force_adj_lists", [None, "directed", "undirected", "both"])
+@pytest.mark.parametrize("force_adj_lists", [None, "homogeneous", "by_other_node_type"])
```

Comment:
This is dramatically changing what this benchmark is measuring; previously it was capturing the cost of using a graph in undirected mode or directed mode (or neither (adjacency matrix only), or both), now it's only capturing the cost of using a graph in either:

- adjacency matrix only
- both undirected and directed mode for homogeneous
- both undirected and directed mode for other_node_type

This benchmark is kinda completely exploding in terms of options, but maybe it instead makes sense to do this as a new parameter like:

```python
@pytest.mark.parametrize("adj_list_directness", ["homogeneous", "by_other_node_type"])
```

Potentially with a:

```python
if force_adj_lists is None and adj_list_directness != "homogeneous":
    pytest.skip("only measure force_adj_lists=None once")
```

However, maybe it's not entirely useful to know the specific cost of the undirected vs. directed adjacencies.